src/mpid/ch4/netmod/ofi/ofi_rma.h

```diff
@@ -118,6 +118,8 @@ static inline void MPIDI_OFI_query_acc_atomic_support(MPI_Datatype dt, int query
 
     dt_index = MPIDI_OFI_DATATYPE(dt_ptr).index;
     op_index = MPIDI_OFI_get_mpi_acc_op_index(op);
+    MPIR_Assert(op_index >= 0);
+    MPIR_Assert(op_index < MPIDI_OFI_OP_SIZES);
 
```

Comment:
Can we use one assert instead of two (`MPIR_Assert(op_index >= 0 && op_index < MPIDI_OFI_OP_SIZES)`), for code simplicity ?

Reply:
I don't agree.  If `assert(condA && condB);` fails, it doesn't tell which condition was actually not satisfied. On the other hand,
```C
assert(condA);
assert(condB);
```
tells you which one failed.

Reply:
In my opinion, these two conditions can be covered by one assert as both are range check. I think this is just personal code preference. I am OK with the current code if you want to keep it.